tools/httpstan/Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: docker-build
+
+TAG := 4.10.1
```

Comment:
Sorry, for the small details. 
Can you move this var to be above the `.PHONY: docker-build`?

```sh
TAG := 4.10.1

.PHONY: docker-build
docker-build:
  ...
```

Reply:
Yes, `TAG` should be top var of the Makefile